xml/installation-installation-install_caasp_heat_templates.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0"?>
+<!DOCTYPE chapter [
+ <!ENTITY % entities SYSTEM "entities.ent"> %entities;
+]>
+<chapter xml:id="install_heat_templates"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Installation of &caasp; &o_orch; Templates</title>
+ <para>
+  This page describes how to install &caasp; &o_orch; template on &productname;.
+ </para>
+ <section xml:id="sec.heat.templates.install">
+  <title>&caasp; &o_orch; Installation Procedure</title>
+   <procedure>
+    <title>Preparation</title>
+    <step>
+     <para>
+          Download the latest &caasp; image (for example,
```

Comment:
Maybe use "\&caasp; for \&ostack; image" instead, as there are CaaSP images for Hyper-V/KVM/Xen as well.